api/src/s2-api.ts

```diff
@@ -120,11 +120,7 @@ export function getPaperUncached(s2Id: string, apiKey?: string): AxiosPromise<Pa
   if (apiKey) {
     conf.headers['x-api-key'] = apiKey;
   }
-  const apiOrigin = (
```

Comment:
I wouldn't change anything in the `api` directory. I don't have a great idea of what goes on here, and we (reader data side) don't really maintain it - we just focus on the data-processing dir.

Reply:
k, i'll revert this change!

Reply:
OR... maybe @mjlangan can shed some light here?

Reply:
No idea what that's used for.

Reply:
Michael says:
<img width="777" alt="Screen Shot 2023-03-28 at 3 20 45 PM" src="https://user-images.githubusercontent.com/47910008/228379663-7c3435ae-c1b0-4226-bfee-1c4b778b39ab.png">